codec-http2/src/main/java/io/netty/handler/codec/http2/internal/hpack/HuffmanDecoder.java

```diff
@@ -213,4 +138,114 @@ private static void insert(Node root, int symbol, int code, byte length) {
             current.children[i] = terminal;
         }
     }
+
+    private static final class DecoderProcessor implements ByteProcessor {
+        // TODO: Is this a good size ?
+        private static final int INITIAL_SIZE = 64;
```

Comment:
@normanmaurer - Also I think this was previously `32` (via default constructor of `ByteArrayOutputStream`). Now that the `byte[]` may live longer ... perhaps we should start with the `32` as a default...and do some followup investigation to see if this is a good default?